src/hotspot/os/bsd/os_perf_bsd.cpp

```diff
@@ -666,7 +678,7 @@ int SystemProcessInterface::SystemProcesses::system_processes(SystemProcess** sy
   *system_processes = next;
 
   return OS_OK;
-#elif defined(__FreeBSD__)
+#elif defined(__FreeBSD__) && !defined(__arm__)
```

Comment:
This code should be machine independent, why are we avoiding it on arm32?

Reply:
some leftover, I'm struggling with git